src/abTestingInfra/ExperimentContext/index.tsx

```diff
@@ -0,0 +1,4 @@
+import React from 'react';
```

Comment:
What is the difference between this and `ABTestingContext`, above?

Reply:
`ExperimentContext` provides `ExperimentName` and `win` function to the `Variant`, while `ABTestingContext` is providing `variant` for the `Experiment`